### PR TITLE
Initial UBI9 images for EAP 8.1

### PR DESCRIPTION
**Pull request by jfdenise**
JDK17 and JDK21 images on UBI9 for EAP 8.1

Behave tests of this ubi9 branch are currently running there, both are green: 
* http://brontes05.lab.eng.brq2.redhat.com/job/imgtest/job/eap8.1JDK17UBI9images/3/
* http://brontes05.lab.eng.brq2.redhat.com/job/imgtest/job/eap8.1JDK21UBI9images/5/

Still some references to osbs, we will replace that with konflux integration

### CODE REVIEW

File: runtime-image/image.yaml [hunk 1]
```diff
@@ -66,6 +73,8 @@ packages:
           - jq
           - vim-minimal
           - unzip
+          # required to allow to use oc/kubectl cp command
+          - tar
```
Comment (jmesnil) [inline runtime-image/image.yaml:77]: 👍  :)

File: runtime-image/rh-jdk17-overrides.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 schema_version: 1
 
 packages:
-  content_sets_file: content_sets_rhel8.yml
+      content_sets_file: content_sets_rhel9.yml
```
Comment (jmesnil) [inline runtime-image/rh-jdk17-overrides.yaml]: indentation looks suspicious... Should it be 2 spaces instead?

File: runtime-image/rh-jdk21-overrides.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 schema_version: 1
 
 packages:
-  content_sets_file: content_sets_rhel9.yml
+      content_sets_file: content_sets_rhel9.yml
```
Comment (jmesnil) [inline runtime-image/rh-jdk21-overrides.yaml]: idem

**Review by jmesnil — COMMENTED** [review]
LGTM but I have a doubt on the content_sets in the YAML file?

File: runtime-image/rh-jdk17-overrides.yaml (hunk 2 above)
Comment (jfdenise) [reply runtime-image/rh-jdk17-overrides.yaml]: Fixed, thanks.

File: runtime-image/rh-jdk21-overrides.yaml (hunk 3 above)
Comment (jfdenise) [reply runtime-image/rh-jdk21-overrides.yaml]: Fixed thanks.

**Comment by jfdenise** [comment]
@jmesnil , this should be good to merge.

**Review by jmesnil — APPROVED** [review]

**Comment by jmesnil** [comment]
@jfdenise thanks!

**Pull request merged**